helm/values.staging.yaml

```diff
@@ -77,3 +77,5 @@ environment:
   NEW_RELIC_APP_NAME: Cranecloud-api-staging
   NEW_RELIC_LICENSE_KEY: ${{ NEW_RELIC_LICENSE_KEY }}
   PROMETHEUS_URL: http://staging-prom.cranecloud.io/
+  MIRA_BACKEND_URL: http://mira.cranecloud.io:5000
```

Comment:
Is Mira intended for internal use only, or will endusers too be accessing it over custom port numbers?
Not related to this PR, but ...

Reply:
Let us add making that experience better to our roadmap, basically make it accessible over the default HTTPs port.